src/devices/bus/isa/svga_cirrus.cpp

```diff
@@ -11,13 +11,130 @@
+    Kelvin 64 / EZ - Source: EPROM dump, Pulled from a non-working card - ROM: CL-GD543x VGA BIOS Version 1.00a - OSC: 14.31818 MHz
+    ROM_SYSTEM_BIOS(1, "kel64ez", "Orchid Kelvin 64 EZ VLB")
+    ROMX_LOAD("ogvlb.bin", 0x08000, 0x8000, CRC(16af65bb) SHA1(16db72ebc67f5dea81418cf25e562eb6da00218c), ROM_BIOS(1) )
+    // Orchid Kelvin 64 MPEG - 16bit VLB - Chips: Cirrus Logic CL-GD5434, Alliance ProMotion-6410, Analog Devices ADSP-2105 KP-40 ED/AA0888-1.1, MUSIC MU9C4870-80DC
+    // ROM: Kelvin 64-VLB BIOS Ver 1.10B - OSC: 14.318180 Mhz, 55.000 MHz, unreadable - Connectors: DB15, 2x3.5mm jacks - Potentiometer on rear bracket
+    ROM_SYSTEM_BIOS(2, "kelmpeg", "Orchid Kelvin MPEG Version 1.10B")
+    ROMX_LOAD("kelvin64_mpeg.bin", 0x00000, 0x10000, CRC(403020a4) SHA1(1ee70f6cc3c7b9a9afb402a1bc939f9f2e0cc739), ROM_BIOS(2) )
+ROM_END
+
+
+
+*/
+
+/*
+    Cirrus Logic VL-GD5436 - PCI cards
+
+//Chip: CL-GD5436-I-QC-A - ROM: CL-GD5436 VL/PCI VGA BIOS Version 1.00b - RAM: 1MB, up to 4MB - Connector: DB15 - VESA connector - OSC: 14.31818MHz
+// Jumpers: JG1 Green Control, IRQ9 enable/disable
+ROM_START( clgd5436 )
+    ROM_REGION(0x08000, "clgd5436", 0)
+    ROM_LOAD("5436.vbi", 0x00000, 0x08000, CRC(7737d03e) SHA1(e4d0b4e7262887dc5a6473ea8909fdc13a6a02c1) )
+ROM_END
+
+*/
+
+/*
+    Cirrus Logic CL-GD5440 - PCI cards
+
+// Chip: Cirrus Logic CL-GD5440-J-QC-B - ROM: CL-GD5440 VGA BIOS Version 1.07 - RAM: 1MB, max. 2MB - OSC:14.31818MHz - Connector: DB15 - VESA connector
+ROM_START( clgd5440 )
+    ROM_REGION(0x10000, "clgd5440", 0)
+    ROM_LOAD("bios.bin", 0x00000, 0x10000, CRC(f0d3d0b4) SHA1(620b0727a20b127f5f32576ec54fbc6f9f437ad3) )
+ROM_END
+*/
+
+/*
+    Cirrus Logic CL-GD5446 - PCI cards
+// Chip:CL-GD5446-HC-A - ROM: CL-GD5436/46 PCI VGA BIOS Version 1.25 - RAM: 1MB, 2MB, 4MB - OSC: 14.31818MHz - Connector: DB15 - VESA feature connector
+ROM_START( clgd5446 )
+    ROM_REGION(0x10000, "clgd5446", 0)
+    ROM_LOAD("5446bv.vbi", 0x00000, 0x10000, CRC(7a859659) SHA1(ff667218261969c48082ec12aa91088a01b0cb2a) )
+ROM_END
+
+*/
+
+/*
+    Cirrus Logic CL-GD5464 Laguna3D
+// DSystems Wizar3D PCI graphics card - Chip: CL-GD5464-HC-A - ROM: DSystems Wizar3D 144a.10H - RAM: 2MB, 4MB, 8MB - Connector: DB15 - VESA feature connector
+ROM_START( clgd5464 )
+    ROM_REGION(0x08000, "clgd5464", 0)
+    ROM_LOAD("dsystems_wizard3d.vbi", 0x00000, 0x08000, CRC(df9f1570), SHA1(4e611f4039b851fd4237d450e38c9d764920a747) )
+ROM_END
+
+*/
+
+/* Cirrus Logic CL-GD5465 Laguna3D
+// Chaintech GA-5465AS AGP graphics card - Chip: CL-GD5465 - ROM: CL-GD546x Laguna PCI VGA BIOS Version 1.62c - RAM: 4MB, 8MB - OSC: 14.3C7Y - Connector: DB15 - VESA feature connector
+ROM_START( clgd5465 )
+    ROM_REGION(0x08000, "clgd5465", 0)
+    ROM_LOAD("chaintech.vbi", 0x00000, 0x08000, CRC(8afa1afb) SHA1(251a953d442dc34738f80371cfbd0fd9f1097635) )
+ROM_END
```

Comment:
PCI roms definitely don't belong to this file, also the highlighted ROM has been added to following file:

https://github.com/mamedev/mame/blob/3ae94acb1f08ffb94cd02574c5104fa3a720736c/src/devices/video/clgd546x_laguna.cpp#L33C1-L37C8

Reply:
So in case it wasn't clear, what I'm asking here is to move these comments to the end of the Laguna file. 
I'll deal with adding the actual PCI boards at some point in the future (i.e. when I'm willing to tackle the full family tree from the ground up) , and having these ROM sets already compiled helps there.